charts/artifact-hub/templates/trivy_service.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: trivy
+  name: {{ include "chart.resourceNamePrefix" . }}trivy
```

Comment:
The `trivyURL` set  in the `scanner` configuration will probably need to be aware of the prefix as well. At the moment the default is `http://trivy:8081`, but when using the prefix the dns name assigned will be different. This value is also mentioned in the values schema documentation.

Reply:
Thanks for the hint! I currently don't have chart metadata for vulnerability scans. I will fix that and test the scanner-trivy communication with the ArtifactHub repo in my deployment.
Concerning general testing: I tested creating three different environments in parallel in a single namespace, including persistent volume for postgres and trivy. However, I am using a route defined externally instead of the ingress of the Chart. I also added a new values file to the CI folder, so the helm/chart-testing action helped me when developing the PR to find deploy-time errors.

Reply:
You can also add temporarily any of the repositories available in the `artifacthub.io` deployment to yours, there are quite a few with images available (including the Artifact Hub one!).

The testing looks great, thanks 👍

Reply:
I tested the vulnerability scans in 4 environments, all in one kubernetes namespace:
- 1 environment with the dynamic resource names disabled (default)
- 3 environments with the dynamic resource names enabled
All with the official artifacthub repository.
The vulnerability scanning works across all environments.
The scanner directly writes the results in the database, so there is no HTTP communication between the hub and the scanner, right?

Reply:
Cool 👍 

>   The scanner directly writes the results in the database, so there is no HTTP communication between the hub and the scanner, right?

Yes, that's right.